src/index.ts

```diff
@@ -1,39 +1,56 @@
-import path from "path";
+import path from 'path';
 import cors from 'cors';
 import express from 'express';
+import { FetchError } from 'node-fetch';
```

Comment:
How about AbortError? https://github.com/node-fetch/node-fetch/blob/master/README.md#class-aborterror

Reply:
I couldn't see any use of [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) in grapql-request library and we don't use it too ( for now ), later we can expand it based on our needs.